base_map/base_map_manager.c

```diff
@@ -59,14 +61,11 @@ void *draw_all(void *arg)
   blue_bomber_sprite(arg);
    if (gi != NULL) {
     printf("\n\n\n\n\nNB CLIENT %d\n\n\n\n\n", gi->nb_client);
-    if (gi->nb_client >= 1)
-     draw_player_1(arg);
-    if (gi->nb_client >= 2)
-     draw_player_2(arg);
-    if (gi->nb_client >= 3)
-     draw_player_3(arg);
-    if (gi->nb_client >= 4)
-     draw_player_4(arg);
+    for (i = 0; i < 4; i++)
```

Comment:
The ifs began at 1 and you begin at 0, I'm not sure it does the exact same thing

Reply:
parce que avant on commencait les players à 1 pour que ce soit "humain" alros que ça saoulait juste